### ISSUE TITLE
Y020: False positive with string default

### ISSUE THREAD

**Issue by srittau** [PyCQA/flake8-pyi#327]
See python/typeshed#9546 for reference. Y020 is triggered on the following  definition:

```python
from typing_extensions import Literal

def foo(bar: str = "x") -> None: ...
```

An empty default string does not trigger the warning.

**Comment by JelleZijlstra** [comment]
Should we do a release soon? You made a bunch of improvements too; unless you have more in the works I think it's time.

**Comment by AlexWaygood** [comment]
> Should we do a release soon? You made a bunch of improvements too; unless you have more in the works I think it's time.

I was just about to post and ask if anybody had anything else they wanted to be included in the next release; otherwise I think it's time as well :)
